src/common/Constants.ts

```diff
@@ -302,13 +320,15 @@ export const SUPPORTED_TOKENS: { [chainId: number]: string[] } = {
   [CHAIN_IDs.MODE]: ["ETH", "WETH", "USDC", "USDT", "WBTC"],
   [CHAIN_IDs.ARBITRUM]: ["USDC", "USDT", "WETH", "DAI", "WBTC", "UMA", "BAL", "ACX", "POOL"],
   [CHAIN_IDs.LINEA]: ["USDC", "USDT", "WETH", "WBTC", "DAI"],
+  [CHAIN_IDs.BLAST]: ["USDC", "USDT", "WETH"],
```

Comment:
Blast maps USDT, DAI and USDC to USDB, which has 18 decimals. For that reason I think we're only planning on supporting DAI for Blast.

```suggestion
  [CHAIN_IDs.BLAST]: ["DAI", "USDT", "WETH"],
```

Reply:
I only removed USDC. I'm guessing I also need to remove USDT?

Reply:
Ah, yeah. We plan to support DAI, WETH and WBTC.